CHANGELOG.md

```diff
@@ -1,3 +1,268 @@
+| ---------------------------------------- | ------------------- | ---------------- | -------------------------------- |
+| "/sync/dag"                              | GET *1              | dag.Info         | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | GET *2              | []byte           | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | POST                | dag.Manifest     | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PUT                 | --               | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PATCH               | stream?          | remote.DsyncHTTPHandler          |
+| "/sync/logs"                             | GET                 | --               | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | PUT                 | []byte           | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | DELETE              | --               | remote.LogbookHTTPHandlery       |
+| "/sync/resolve"                          | GET                 | DSRef            | remote.RefsHTTPHandler           |
+| "/sync/resolve"                          | DELETE              | --               | remote.RefsHTTPHandler           |
+|                                          |                     |                  |                                  |
+
+
+## Redesigned lib interface
+In general, we've streamlined the core functionality and reconciled input params in the `lib` package (which contain the methods and params that power both the api and cmd), so that no matter how you are accessing functionality in qri, whether using `lib` as a package,  using the api, or using the command line, you will have, not only consistent input parameters, but also consistent behavior. We're also utilizing our new `dispatch` pattern to replace our old rpc client with the same http json api that we expose to our users. That way all our api's, http or otherwise, have the same expectations. If something is broke in one place, it is broken in all places, consequently, when it is fixed in one place it will be fixed in all places.
+
+These changes will also help us in our upcoming challenges to refine expand the notion of identity inside of Qri.
+
+## Step-Based Transform Execution
+We've changed the underlying way that transforms execute, dividing the work into steps, like `Setup`, `Download`, and `Transform`. With this change, we are able to report the status of their transform back to the user. It will also give the user a better sense of what is happening in their transform: how long did the `Download` step take, which step failed, which steps worked. It also paves the way for a better transform development environment in the future. Users will be able to re-run certain steps or cache steps that they don't want to waste time re-running.
```

Comment:
I don't think this feature is quite ready to talk about. Yes, it's in Qri, but at this point I'd label it _pre-experimental_. Let's remove this, and save this exact paragraph for the next set of release notes